utils/numberToOrdinal.ts

```diff
@@ -7,13 +7,13 @@ export function numberToOrdinal(num: number): string {
   const k = num % 100
 
   if (j === 1 && k !== 11) {
-    return `${num}st`
+    return `${num.toLocaleString()}st`
   }
   if (j == 2 && k != 12) {
-    return `${num}nd`
+    return `${num.toLocaleString()}nd`
   }
   if (j == 3 && k != 13) {
-    return `${num}rd`
+    return `${num.toLocaleString()}rd`
   }
-  return `${num}th`
+  return `${num.toLocaleString()}th`
```

Comment:
I could be wrong here but I don't think this is correct, largely because `toLocaleString` is locale / language specific, but our ordinal system function is always gonna return English-specific suffixes.

I don't think a formatted number like `8,400th` is correct, in the same way that in Germany the same number shouldn't be `8.400th`

Reply:
hmm, you might be right here. Instinctively, I felt like the 5 digit numbers needed a comma, even for the ordinal values. 

I'll revert that commit for now.